### DISCUSSION TITLE
Search for more specific AWS unused policies

### DISCUSSION THREAD

**Discussion by haiauth** [cloud-custodian/cloud-custodian#7447] [AWS]
### Ask your question

Is there anyway to search for AWS user-created, customer-managed, or user-created inline policies? 

### Policy

```shell
policies:
  - name: iam-policy-unused
    comment: | 
      Check for any policy that is not being used.
    resource: iam-policy
    filters:
      - type: unused
```

### Relevant log/traceback output

```shell
This is the policy that I am currently using but it is too generic and it returns too many unnecessary outputs (include AWS unused policies). I know that is what it is supposed to do but I am trying to find a way to filter for policies that are created by users but not currently in use or attached to any role or user.
```

**Comment by ajkerrigan** [comment]
As of 0.9.13 we [changed the default behavior](https://github.com/cloud-custodian/cloud-custodian/pull/6828) to only fetch local (user-managed rather than AWS-managed) policies. That change was motivated by exactly this sort of use case - including AWS-managed policies tends to be slow, noisy and not useful.

Which version of custodian are you running?

**Comment by haiauth** [comment]
Custodian:   0.9.14
